src/modules/apps/e2e/report.tsx

```diff
@@ -0,0 +1,270 @@
+import * as React from 'react'
+import { Box, Color, Text, Static } from 'ink'
+import { all, difference, groupBy, toPairs, values, countBy, sum, pathOr, path } from 'ramda'
+import { parseAppId } from '@vtex/api'
+
+import { SpecReport, AppReport, TestReport, SpecTestReport, Screenshot } from '../../../clients/Tester'
+import { useInterval } from './useInterval'
+
+type AppIdProps = {
+  appId: string
+}
+
+const AppId: React.FunctionComponent<AppIdProps> = ({ appId }) => {
+  const { name, version, build } = parseAppId(appId)
+  const [vendor, app] = name.split('.')
+  return (
+    <Box>
+      <Color blue>{vendor}</Color>
+      <Color dim>.</Color>
+      {`${app}@${version}`}
+      {build && <Color dim>{`-${build}`}</Color>}
```

Comment:
That's the way we usually represent linked apps: `vtex.toolbelt-telemetry@0.1.0+build1585346569`
```suggestion
      {build && <Color dim>{`+${build}`}</Color>}
```